src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> sutNumbers;
+
+    @BeforeEach
+    void setUp() {
+        sutNumbers = new HashSet<>();
+        sutNumbers.add(1);
+        sutNumbers.add(1);
+        sutNumbers.add(2);
+        sutNumbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size 메소드를 수행할 경우 numbers의 크기가 출력되어야한다.")
+    public void SizeMethod_Should_Return_NumberSize() throws Exception{
+        //given
+        //when
+        int size = sutNumbers.size();
+        int expectSize = 3;
+        //then
+        assertThat(size).isEqualTo(expectSize);
```

Comment:
size를 꺼내서 사용하는것도 좋지만

```suggestion
        assertThat(sutNumbers).hasSize(3);
```

이렇게 assertj api 를 사용하는 방법도 있어요! 😁

Reply:
새로운 방법 알아가네요.
감사합니다!